### DISCUSSION TITLE
QSO Start / end times

### DISCUSSION THREAD

**Discussion by Craig3FW** [magicbug/Cloudlog#2515] [Q&A] [answered]
How do you log different start and finish times for a QSO?
In most SW I've used once the Call window loses focus and you're in the "body" of the QSO entry window the start time is set.
The end time is then set when the QSO ends.

This doesn't happen with Cloudlog, when you look at the QSO both times are in fact the end time.

This is irrelevant for most sat qso's and some digimode qso's but is a factor on HF.

As background EI regs require logging start and finish times except for mobile/portable operation.
Therefore I cannot use Cloudlog as my main log until I can figure this one out.

Any advice / help would be appreciated.
Best 73
Craig EI3FW

**Comment by magicbug** [comment, marked as answer]
At the moment you can't, but this could be an option so that when you type a callsign in we start that as start time and then when you press add qso its taken as the end time, this would require some rework of the interface code

**Reply by Craig3FW** [reply]
OK, thanks for the info. Guess I'll have to have dual logs for another little while.

**Comment by phl0** [comment]
![Screenshot from 2023-09-28 22-36-57](https://github.com/magicbug/Cloudlog/assets/7112907/d14e913c-2361-4df5-b0d2-4ed00f949272)

Guess it needs some code around this ... :)

**Reply by Craig3FW** [reply]
That's what I need so happy days soon?

**Comment by magicbug** [comment]
@Craig3FW Can confirm this is now in the dev branch and will be landing in the next stable release probably end of this week.

**Reply by Craig3FW** [reply]
Thanks for the update ....... and thanks for the effort
